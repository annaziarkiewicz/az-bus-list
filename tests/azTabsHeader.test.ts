import { describe, test, expect } from "vitest";
import { render } from "@testing-library/vue";

import AzContainer from "@/components/azContainer.vue";

const props = [
  {
    bg: false,
    display: "col",
    outline: false,
  },
  {
    bg: true,
    display: "row",
    outline: true,
  },
];
describe("AzContainer", () => {
  test.each(props)("it should match snapshot", (propsCase) => {
    // Act
    const { container } = render(AzContainer, {
      props: propsCase,
    });

    // Assert
    expect(container).toMatchSnapshot();
  });
});
