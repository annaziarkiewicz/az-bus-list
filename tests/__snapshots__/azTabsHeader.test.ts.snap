// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`AzContainer > it should match snapshot 1`] = `
<div>
  <div
    class="az-container col"
    data-v-eff618bf=""
  >
    
    
  </div>
</div>
`;

exports[`AzContainer > it should match snapshot 2`] = `
<div>
  <div
    class="az-container az-container--bg az-container--outline row"
    data-v-eff618bf=""
  >
    
    
  </div>
</div>
`;
